cpu/cc26x0/include/cpu_conf.h

```diff
@@ -21,6 +21,8 @@
 #ifndef CPU_CONF_H
 #define CPU_CONF_H
 
+#define SET_MODE_CONF_1_ALT_DCDC_IPEAK 0x2 /**< 32 mA */
+#define SET_MODE_CONF_XOSC_FREQ        0x3 /**< 24 MHz xtal */
```

Comment:
Should the board config be able to overwrite those?

Reply:
Not really, AFAIK most of the boards using `cc26x0`/`cc13x0` provide the option to select a 48 MHz crystal, however I haven't seen any out there, and on the [guide] for crystal selection only makes mention on 24 MHz crystal for `cc26x0`/`cc13x0` CPUs, it's the same for `cc26x2`/`cc13x2` as only 48 MHz crystals are used.

However, should I add a `#ifndef` guard for future proofing :eyes: ?

[guide]: https://www.ti.com/lit/an/swra495h/swra495h.pdf?ts=1592499401041

Reply:
If other frequencies are possible, we should allow for custom boards to use them.
Adding an `#ifndef` won’t hurt.